### PR TITLE
Aria Operations Rebranded Release, v1.8.14

### PR DESCRIPTION
**Pull request by ustinov**
Please review the updated `README.md` in a human-readable form here
https://github.com/wavefrontHQ/wavefront-sdk-python/tree/austinov/rebranding

### CODE REVIEW

**Review by oppegard — APPROVED** [review]

File: README.md
Comment (Margarita-Staneva) [inline README.md:23]: VMware Aria Operations for Applications (formerly known as Wavefront) Python SDK lets you send raw data from your Python application to Operations for Applications using a `wavefront_sender` interface. The data is then stored as metrics, histograms, and trace data. This SDK is also referred to as the Wavefront Sender SDK for Python.

File: README.md
Comment (Margarita-Staneva) [inline README.md:25]: Although this library is mostly used by the other Operations for Applications Python SDKs to send data to Operations for Applications, you can also use this SDK directly. For example, you can send data directly from a data store or CSV file to Operations for Applications.

File: README.md
Comment (Margarita-Staneva) [inline README.md:516]: Log in to GitHub, click **Releases** on the right, and click **Draft a new release**.

File: README.md
Comment (Margarita-Staneva) [inline README.md:513]: Search for `version=` in the file to find the version number, for example `1.8.15`.

File: README.md
Comment (Margarita-Staneva) [inline README.md:514]: 4. Create a pull request, get it reviewed and approved, and merge it after approval.

File: README.md
Comment (Margarita-Staneva) [inline README.md:524]: From the GitHub top ...

**Review by Margarita-Staneva — COMMENTED** [review]
The full name is VMware Aria Operations for Applications and the short form is just Operations for Applications. 

I found some other minor things that can be fixed. :)

Btw, are you planning to update other mentions of Wavefront, for example "Wavefront service"? If this is going to happen sometime later, maybe under the first two paragraphs in the readme, you can include something like that:

Note: We're in the process of updating the product name to Operations for Applications, but in many places we still refer to it as Wavefront.

**Comment by ustinov** [comment]
> Btw, are you planning to update other mentions of Wavefront, for example "Wavefront service"? If this is going to happen sometime later, maybe under the first two paragraphs in the readme, you can include something like that:
> 
> Note: We're in the process of updating the product name to Operations for Applications, but in many places we still refer to it as Wavefront.

Thank you @Margarita-Staneva for reviewing the changes! I was actually looking for your help on how far should we go, since we're keeping the code unchanged, I'm keeping Wavefront Sender references unchanged, for example, as we still have the class named `WavefrontSender` in the code, and so on.

**Review by Margarita-Staneva — APPROVED** [review]
lgtm

**Pull request merged**
